test/unit_test/flatbuffer_parser_test.cc

```diff
@@ -71,7 +71,7 @@ BOOST_AUTO_TEST_CASE(check_flatbuffer)
   uint8_t* buf = build._builder.GetBufferPointer();
   int size = build._builder.GetSize();
 
-  all->flat_converter = new VW::parsers::flatbuffer::parser(buf);
+  all->flat_converter = std::unique_ptr<VW::parsers::flatbuffer::parser>(new VW::parsers::flatbuffer::parser(buf));
```

Comment:
You can use `VW::make_unique` to eliminate the need for new

```suggestion
  all->flat_converter = VW::make_unique<VW::parsers::flatbuffer::parser>(buf);
```